openpype/hosts/tvpaint/lib.py

```diff
@@ -510,7 +512,12 @@ def composite_rendered_layers(
     image_size = None
 
     # Generate composited images
+    if export_frames is None:
+        export_frames = []
+
     for frame_index in range(range_start, range_end + 1):
+        if frame_index not in export_frames:
```

Comment:
Ajouter une verif:
`if frame_index not in export_frames and export_frames :`
Sinon, si on ne donne pas de export frame, on ne passe jamais dans la suite du code